Dockerfile

```
@@ -1,5 +1,5 @@
 # Stage 1: Build the frontend assets
-FROM node:14 AS front-end
+FROM node:17 AS front-end
```

```suggestion
FROM node:16 AS front-end
```
node 16 is an LTS release with EOL on 2024-04-30, while node 17 is only supported until 2022-06-01

---

Oh yeah, that's a lot better :)

---

i think you have to accept that, i don't have that button on this PR